packages/aws-appsync/package.json

```diff
@@ -38,7 +38,7 @@
     "uuid": "3.x"
   },
   "peerDependencies": {
-    "@react-native-community/async-storage": "^1.11.0",
+    "@react-native-async-storage/async-storage": "^1.11.0",
     "@react-native-community/netinfo": "^5.0.0"
   }
```

Comment:
Let's remove these `peerDependencies` altogether. See https://github.com/awslabs/aws-mobile-appsync-sdk-js/pull/675 for additional info.